### PR TITLE
fix: no default accounts

### PR DESCRIPTION
**Pull request by weichweich**
## fixes KILTProtocol/ticket#1715

This should not change anything. But just to make sure that these potentially problematic lines are not copied and used somewhere else, I would change it.

## Checklist:

- [x] I have verified that the code works
  - [x] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] This PR does not introduce new custom types
  - [ ] If not, I have opened a companion PR with the type changes in the [KILT types-definitions repository](https://github.com/KILTprotocol/type-definitions/pulls)
- [x] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: primitives/src/constants.rs [hunk 1]
```diff
@@ -219,12 +219,10 @@ pub mod did {
...
 pub mod treasury {
 	use crate::{Balance, BlockNumber};
-	use frame_support::PalletId;
 
 	use super::{BLOCKS_PER_YEAR, KILT};
 
 	pub const INITIAL_PERIOD_LENGTH: BlockNumber = BLOCKS_PER_YEAR.saturating_mul(5);
 	const YEARLY_REWARD: Balance = 2_000_000u128 * KILT;
 	pub const INITIAL_PERIOD_REWARD_PER_BLOCK: Balance = YEARLY_REWARD / (BLOCKS_PER_YEAR as Balance);
-	pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"kilt/tsy");
```
Comment (wischli) [inline primitives/src/constants.rs:229]: Q: Why did you move this out of constants?

File: pallets/kilt-launch/src/mock.rs [hunk 2]
```diff
@@ -111,12 +111,17 @@ parameter_types! {
 	pub const AutoUnlockBound: u32 = 70;
 }
 
+parameter_types! {
+	pub const LaunchPalletId: PalletId = PalletId(*b"kilt/thx");
```
Comment (wischli) [inline pallets/kilt-launch/src/mock.rs:115]: M: Should be
```suggestion
	pub const LaunchPalletId: PalletId = PalletId(*b"kilt/lch");
```

**Review by wischli — COMMENTED** [review]
I would like to move the pallet ids to the constants and import them from there.

File: primitives/src/constants.rs (hunk 1 above)
Comment (weichweich) [reply primitives/src/constants.rs:229]: The IDs should all be at the same place so that you can spot duplicates quickly.
We could have them in the primitives constants. But IMHO this makes it only more complicated. They are mostly used in a single place. It doesn't matter if they differ between peregrine and spiritnet so I don't see the point in moving them to the primitives.

I'm also not sure if it's a good idea to have a fixed prefix like `kilt/`. effectively there are only three letters left to change?

File: pallets/kilt-launch/src/mock.rs (hunk 2 above)
Comment (weichweich) [reply pallets/kilt-launch/src/mock.rs]: fbf0251

File: primitives/src/constants.rs (hunk 1 above)
Comment (weichweich) [reply primitives/src/constants.rs:229]: I moved it back to the primitives.

File: primitives/src/lib.rs [hunk 3]
```diff
@@ -178,3 +151,14 @@ pub type FeeSplit<R, B1, B2> = SplitFeesByRatio<R, FeeSplitRatio, B1, B2>;
 /// https://w3f-research.readthedocs.io/en/latest/polkadot/Token%20Economics.html#-2.-slow-adjusting-mechanism
 pub type SlowAdjustingFeeUpdate<R> =
 	TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
+
+/// Unique identifiers for pallets.
```
Comment (Diiaablo95) [inline primitives/src/lib.rs:155]: W: I would move the pallet_id module into its own file. I know it does not have a lot of information, but it is easier to find. This lib.rs file feels like a big pot of stuff.

**Review by Diiaablo95 — APPROVED** [review]
LGMT! I did not check the crowdloan pallet as I guess we need to merge `master` back into `develop` and `develop` back into this branch.

File: primitives/src/lib.rs (hunk 3 above)
Comment (weichweich) [reply primitives/src/lib.rs]: 7857d6d

File: primitives/src/constants.rs (hunk 1 above)
Comment (weichweich) [reply primitives/src/constants.rs:229]: 143923e

**Review by wischli — APPROVED** [review]
LGTM!
